front/src/common/Map/Settings/MapSettingsBackgroundSwitches.tsx

```diff
@@ -9,23 +21,17 @@ import {
   updateShowOSMtracksections,
   updateTerrain3DExaggeration,
 } from 'reducers/map';
-import SwitchSNCF, { SWITCH_TYPES } from 'common/BootstrapSNCF/SwitchSNCF/SwitchSNCF';
-import iconIGNBDORTHO from 'assets/pictures/mapbuttons/mapstyle-ortho.jpg';
-import iconIGNSCAN25 from 'assets/pictures/mapbuttons/mapstyle-scan25.jpg';
-import iconIGNCadastre from 'assets/pictures/mapbuttons/mapstyle-cadastre.jpg';
-import iconOSM from 'assets/pictures/mapbuttons/mapstyle-normal.jpg';
-import iconOSMTracks from 'assets/pictures/mapbuttons/mapstyle-osm-tracks.jpg';
-import Slider from 'rc-slider';
-import { getTerrain3DExaggeration } from 'reducers/map/selectors';
+import { RootState } from 'reducers';
 
-export default function MapSettingsBackgroundSwitches() {
+const FormatSwitch: FC<{
+  name: string;
+  onChange: SwitchSNCFProps['onChange'];
```

Comment:
You can remove the export of `SwitchSNCFProps`
```suggestion
  onChange: InputHTMLAttributes<HTMLInputElement>['onChange'];
```

Reply:
Like I said in a previous note, I prefer to keep the type of the component itself than the generic HTML one